### PR TITLE
Html css popup

### PR DESCRIPTION
**Pull request by yaripey**
### html-css-popup

[Demo](https://yaripey.github.io/popup-task/) |
[Code base](https://github.com/yaripey/popup-task)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by lenadgit** [comment]
Good job!  especially a triangle in the top of popup)

File: submissions/yaripey/html-css-popup/css/main.css [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+}
+
+.popupmenu-wrap {
+  visibility: hidden;
+  position: absolute;
+  opacity: 0;
+  left: -200px;
+  top: 50px;
+  box-shadow: 0 0 15px rgba(0, 0, 0, 0.3);
+
```
Comment (lenadgit) [inline submissions/yaripey/html-css-popup/css/main.css:134]: could you please remove this empty line

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/yaripey/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  <header>
+    <nav>
+      <a href="#" class="nav-item">Mail</a>
+      <a href="#" class="nav-item">Images</a>
+      <div class="nav-item menu">
+        <label class="menu-button">
+          <input type="checkbox" class="menu-check">
+          <span class="menu-icon"></span>
+          <span class="popupmenu-wrap">
+            <section class="popupmenu">
```
Comment (lenadgit) [inline submissions/yaripey/html-css-popup/index.html:21]: would be better to add markup `nav > ul > li` for dropdown menu

File: submissions/yaripey/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/main.css">
+  <title>Popup task</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <a href="#" class="nav-item">Mail</a>
```
Comment (lenadgit) [inline submissions/yaripey/html-css-popup/index.html:14]: would be better to add the markup `ul > li` for main menu

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/yaripey/html-css-popup/index.html (hunk 3 above)
Comment (yaripey) [reply submissions/yaripey/html-css-popup/index.html:14]: Should it be `header > ul > li`?

File: submissions/yaripey/html-css-popup/index.html (hunk 2 above)
Comment (yaripey) [reply submissions/yaripey/html-css-popup/index.html:21]: You mean instead of `section` should be `nav`?

**Comment by yaripey** [comment]
Implemented the proposed changes.

**Review by lenadgit — APPROVED** [review]
Thank you for the submission. Feel free taking the next course module while you submission is being reviewed

**Pull request merged**